packages/modules/clickhouse/src/clickhouse-container.test.ts

```diff
@@ -0,0 +1,184 @@
+import { ClickHouseClient, createClient } from "@clickhouse/client";
+import { Wait } from "testcontainers";
+import { ClickHouseContainer } from "./clickhouse-container";
+
+interface ClickHouseQueryResponse<T> {
+  data: T[];
+}
+
+describe("ClickHouseContainer", { timeout: 180_000 }, () => {
+  it("should start successfully with default settings", async () => {
```

Comment:
Don't think this test is necessary - these fields are all tested implicitly in the following tests